src/layouts/layouts/SiteViewLayout/SiteViewLayout.tsx

```diff
@@ -50,20 +51,30 @@ export const SiteViewLayout = ({
         >
           <Sidebar />
         </GridItem>
-        <GridItem
-          area="content"
-          as={VStack}
-          p="2rem"
-          spacing="2rem"
-          bgColor="gray.50"
-          w="100%"
-          h="100%"
-          divider={<StackDivider borderColor="border.divider.alt" />}
-          {...rest}
-        >
-          {children}
-        </GridItem>
+        <ContentGridLayout>{children}</ContentGridLayout>
       </Grid>
     </>
   )
 }
+
+export const ContentGridLayout = ({
```

Comment:
nit: could we change the name to `SiteViewContent` so that the grouping w/ `SiteViewLayout` is clearer

Reply:
ac4b18f